src/stravalib/model.py

```diff
@@ -287,6 +292,10 @@ class BackwardCompatibilityMixin:
     return values based on what is defined in the following class attributes:
 
     * _field_conversions
+
+    Notes
+    ------
+    The class attributes below are not yet implemented:
```

Comment:
@jsamoocha i'm doing some cleanup. i noticed this TODO in the notes. is it related to https://github.com/stravalib/stravalib/issues/323 or can you help me understand what the TODO actually is here?

thank you.

Reply:
These TODOs are nice to have, i.e., we can (potentially) use this mixin to warn users when they try to access deprecated or unsupported (by Strava) fields. However, I'm not sure we'll still use this mixin when moving to Pydantic2, and I am also unsure if it will be easy and straightforward to implement that logic. Feel free to remove these TODOs.

Reply:
ok thank you!